src/React/DOM/Props.purs

```diff
@@ -168,9 +177,27 @@ icon = unsafeMkProps "icon"
 _id :: String -> Props
 _id = unsafeMkProps "id"
 
+inputMode :: String -> Props
+inputMode = unsafeMkProps "inputMode"
+
+integrity :: String -> Props
+integrity = unsafeMkProps "integrity"
+
+is :: String -> Props
```

Comment:
I don't believe that this is an HTML element. From what I have seen it pertains to polymer. Should this be included? Can someone be using polymer with React? If so, perhaps such attributes should be in a separate module?